fvm/src/executor/default.rs

```diff
@@ -123,17 +124,52 @@ where
             });
 
             let result = cm.with_transaction(false, |cm| {
+                let pl = cm.context().price_list;
+
+                if let Some(sender_state) = sender_st {
```

Comment:
note: we can probably afford to lookup the sender state again. I'd consider just returning some form of "sender needs to be created" flag, or something like that?

Reply:
I don't have a strong preference. My concern is that we might insert some other mutation between there and here, but if you don't think that's an issue, I have no strong objections.